be/src/udf/record_store_impl.cpp

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "runtime/free_pool.hpp"
+#include "runtime/runtime_state.h"
+#include "runtime/descriptors.h"
+#include "udf/udf_internal.h"
+
+namespace doris {
+
+doris_udf::RecordStore* RecordStoreImpl::create_record_store(FreePool* free_pool, TupleDescriptor* descriptor) {
```

Comment:
MemPool is preferred. Because in execution engine all tuple is allocate from MemPool.